{"name": "test_login_without_providing_credentials", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('log_on_failure', <FixtureRequest for <Function test_login_without_providing_credentials>>)", "trace": "file /Users/anashuhada/PycharmProjects/SeleniumWithPython/tutorialsninja/test_Login.py, line 19\n      def test_login_without_providing_credentials(self):\nE       fixture 'log_on_failure' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, capture_failure_screenshot, doctest_namespace, extra, extras, include_metadata_in_junit_xml, metadata, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, setup_and_teardown, testrun_uid, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/anashuhada/PycharmProjects/SeleniumWithPython/tutorialsninja/test_Login.py:19"}, "attachments": [{"name": "log", "source": "4edfc008-04f8-41cf-bb5a-3d901d074797-attachment.txt", "type": "text/plain"}], "start": 1748582208766, "stop": 1748582208766, "uuid": "e49637f0-fa54-4602-ac5d-c35f9c29af1c", "historyId": "b9684dcc0bc038667179ae30bfd795f6", "testCaseId": "b9684dcc0bc038667179ae30bfd795f6", "fullName": "tutorialsninja.test_Login.TestLogin#test_login_without_providing_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "tutorialsninja"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Macmini"}, {"name": "thread", "value": "5494-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tutorialsninja.test_Login"}]}